app/ui-flextab/client/tabs/createInviteLink.js

```diff
@@ -0,0 +1,92 @@
+import { ReactiveVar } from 'meteor/reactive-var';
+import { Template } from 'meteor/templating';
+import toastr from 'toastr';
+import s from 'underscore.string';
+
+import { t, APIClient } from '../../../utils';
+
+function getInviteLink(instance, rid, days, maxUses) {
+	APIClient.v1.post('findOrCreateInvite', {
+		rid,
+		days,
+		maxUses,
+	}).then((result) => {
+		if (!result) {
+			toastr.error(t('Failed_to_generate_invite_link'));
+			return;
+		}
+
+		instance.inviteData.set(result);
+		instance.url.set(result.url);
+		instance.isEditing.set(false);
+	}).catch(() => {
+		toastr.error(t('Failed_to_generate_invite_link'));
+	});
+}
+
+Template.createInviteLink.helpers({
+	isEditing() {
+		return Template.instance().isEditing.get();
+	},
+
+	isReady() {
+		return Template.instance().isReady.get();
+	},
+
+	url() {
+		return Template.instance().url.get();
+	},
+
+	linkExpirationText() {
+		const data = Template.instance().inviteData.get();
+
+		if (!data) {
+			return '';
+		}
+
+		if (data.expires) {
+			const expiration = new Date(data.expires);
+
+			if (data.maxUses) {
+				const usesLeft = data.maxUses - data.uses;
+				return t('Your_invite_link_will_expire_on_{date}_or_after_{usesLeft}_uses').replace('{date}', expiration.toLocaleDateString()).replace('{usesLeft}', usesLeft);
```

Comment:
Please use the native params of the translation api to pass data

Reply:
We have methods to format dates based on the user’s and server’s settings as well